packages/terra-avatar/translations/en.json

```diff
@@ -0,0 +1,3 @@
+{
+    "Terra.avatar.deceased": "Deceased"
+}
```

Comment:
You need to log a translations JIRA for this entry to be translated into other terra supported languages as well.

Reply:
Logged, here - https://jira.cerner.com/browse/TRANS-14113